api/internal/core/store/validate.go

```diff
@@ -165,12 +165,6 @@ func checkUpstream(upstream *entity.UpstreamDef) error {
 		return nil
 	}
 
-	if upstream.PassHost == "node" && upstream.Nodes != nil {
-		if nodes := entity.NodesFormat(upstream.Nodes); len(nodes.([]*entity.Node)) != 1 {
-			return fmt.Errorf("only support single node for `node` mode currently")
-		}
-	}
```

Comment:
@guoqqqi Need to adjust the change.

Reply:
should not remove this block of code. see: https://github.com/apache/apisix/blob/master/apisix/upstream.lua#L397-L401

Reply:
fixed